### DISCUSSION TITLE
Save button save empty values

### DISCUSSION THREAD

**Discussion by cyber-kivi** [Budibase/budibase#7230] [Help] [answered]
Hello!
I have field "select value" to update value in main table with save Button. But it save empty row instead opdate only one value. It must save new value in existing column at main table. How to correct it?

![image](https://user-images.githubusercontent.com/106427850/184140680-e27ff725-75fe-41f3-be9f-6762d105931e.png)

Form settings - update
There are two data providers:
1) Main - main table
2) Values - values for column in main table

**Comment by melohagan** [comment, marked as answer]
Hi @cyber-kivi 

If you want to update a row, you will need to provide an id. You can do that by clicking on *Add column* and selecting **_id**.

If the *Main Form* was nested within a repeater relating to the 'Contraindictions' table, then Budibase would automatically pick up the id field for you, so you wouldn't need to add the _id column.
